docs/spec/spec.md

```diff
@@ -1258,6 +1274,12 @@ service on new graphql:Listener(4000) {
 ```
 > **Note:** The field enable accepts a `boolean` that denotes whether the client is enabled or not. By default, it has been set to `false`. The optional field `path` accepts a valid `string` for the GraphiQL service. If the path is not given in the configuration, `/graphiql` is set as the default path.
 
+#####  9.1.5.1 GraphiQL for Subscriptions
+GraphiQL uses WebSockets as the communication protocol for Subscription requests. And there is an underlying sub protocol called `graphql-transport-ws` which follows a standard message structure for subscriptions defined in the [specification](https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md). Since GraphiQL follows the same sub protocol, all the subscription requests/responses will be wrapped in this format.
+
+> **Note:** There is another supported sub protocol named `graphql-ws`. GraphQL playground follows that sub protocol for subscriptions.  
+> If a subscription request occurs without a sub-protocol, responses will be dispatched according to the usual behavior. 
+
```

Comment:
This information about the protocols should come under the section `2.1 Listener`. There's not point of having them here.